core-bundle/src/Resources/views/Image/Studio/_macros.html.twig

```diff
@@ -135,7 +135,7 @@
             {% set caption_attributes = figure.options.caption_attr|default({})|merge(options.caption_attr|default({})) %}
             {% set base_attributes = { 'itemprop': 'caption' } %}
             <figcaption{{ _self.html_attributes(base_attributes|merge(caption_attributes)) }}>
-                {{- figure.metadata.caption -}}
+                {{- figure.metadata.caption|raw -}}
```

Comment:
The twig macro doesn’t have to be backwards compatible, so I would not add this here.

Reply:
Not sure about this. I think I'd prefer to drop the functionality altogether or be consistent.

Reply:
+1 for consistency

Reply:
As discussed in Mumble on January 14th, we want to keep the `|raw` flag.